ogr/ogrsf_frmts/vfk/vfkfeature.cpp

```diff
@@ -86,6 +86,118 @@ void IVFKFeature::SetFID(GIntBig nFID)
     }
 }
 
+/*!
+  \brief Compute determinant of matrix with columns x,y and z
+
+  Simple formula requires reasonable numbers
+
+  \param x first column array
+  \param y second column array
+  \param z third column array
+
+  \return double determinant value
+*/
+double IVFKFeature::GetDeterminatOfMatrixDim3(double x[3], double y[3],
+                                              double z[3])
+{
+    return x[0] * y[1] * z[2] - x[0] * z[1] * y[2] - y[0] * x[1] * z[2] +
+           y[0] * x[2] * z[1] + z[0] * x[1] * y[2] - z[0] * y[1] * x[2];
+}
+
+/*!
+  \brief Find circle center determined by three point
+
+  \param c_xy circle center coordinates array
+  \param x array of three x coordinates
+  \param y array of three x coordinates
+*/
+void IVFKFeature::GetCircleCenterFrom3Points(double c_xy[2], double x[3],
```

Comment:
@seidlmic could you check if the OGRGeometryFactory::GetCurveParameters() couldn't be used instead ? That would save code duplication